application/frontend/web/load-app.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+require(__DIR__ . '/../../vendor/autoload.php'); //TODO: why is this required for the web app but not the tests?
```

Comment:
Are you referring to PHPUnit tests? If so, I think it's this: By the time the tests are run, the bootstrap file for PHPUnit has already pulled in the autoloader (otherwise it couldn't have loaded the classes used for running the tests). For web access, this is the first file run, so any use of classes needs to have the autoloader already in place.